### PR TITLE
Refactored Code and file structure

### PR DESCRIPTION
**Pull request by varun1233**
1. Refactored Code - naming and spaces wherever required
2. Corrected File Structure
3. Remove Commented code
4. Added Usage File

### CODE REVIEW

**Review by Ankur9669 — APPROVED** [review]
The changes look good to me. Great Work!!

File: components/alert/alert.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+            ><i class="fab fa-github"></i
+          ></a>
+          <a
+            class="h3"
+            href="https://twitter.com/VarunMittal1233"
+            target="_blank"
+            ><i class="fab fa-twitter"></i
+          ></a>
+          <a
+            class="h3"
```
Comment (Adarsh-Patel-Dev) [inline components/alert/alert.html:76]: Instead of "h3" try to use descriptive class name

**Review by Adarsh-Patel-Dev — APPROVED** [review]
No Changes needed
Your code structure is very clean.
You can merged.

File: components/alert/alert.html (hunk 1 above)
Comment (varun1233) [reply components/alert/alert.html:76]: Actually I have used spacing font size for element h3 and class h3 that is why using this

**Pull request merged**